.github/workflows/build_and_test.yml

```diff
@@ -91,6 +91,49 @@ jobs:
         cmake --build . --config Debug
         ctest -C Debug --output-on-failure
 
+  simdUbuntu:
+    name: "SIMD programming using C++ intrinsic functions on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-22.04
+        cxx:
+          - clang++
+    env:
+      ORC_USER_SIMD_LEVEL: avx512
```

Comment:
Is it case-insensitive? I prefer uppercase here.

Reply:
No, it is not case-insensitive.
I will change it to uppercase here.

Reply:
Done